wireapp/wire-ios-mono
# fix: add post migration to trigger sync resources - WPB-14794

### Issue

Following the issue with migration of federation backend, we need to sync resources (users, conversations) to get the domain set correctly.

Solution:

Create a db migration that will set a flag in the persistent store so we can trigger a resyncResources.

* Db with new Entity (CoreData will not run a migration without a datamodel change, version is not enough)  
* reset the flag in didFinishSlowSync

### Testing

1. login with 3.114.0
2. update to 3.114.1
3. notice a sync resouces (slow sync) is performed
---

### Checklist


---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
